PayPal/src/test/java/com/braintreepayments/api/paypal/PayPalVaultRequestUnitTest.java

```diff
@@ -132,6 +132,14 @@ public void parcelsCorrectly() {
         request.setMerchantAccountId("merchant_account_id");
         request.setUserPhoneNumber(new PayPalPhoneNumber("1", "1231231234"));
 
+        RecurringBillingAmountBreakdown amountBreakdown = new RecurringBillingAmountBreakdown(
```

Comment:
I don't think we need any changes to this test file - all these changes should be moved to the `PayPalCheckoutRequestUnitTest` file instead